stable/mattermost-team-edition/README.md

```diff
@@ -90,6 +83,24 @@ Alternatively, a YAML file that specifies the values for the parameters can be p
 $ helm install --name my-release -f values.yaml stable/mattermost-team-edition
 ```
 
+### Example configuration
+
+A basic example of a `.yaml` file with values that could be passed to the `helm`
+command with the `-f` or `--values` flag to get started.
+
+```yaml
+ingress:
+  enabled: true
+  hosts:
+    - mattermost.example.com
+
+configJSON:
```

Comment:
the example you say you can use like a yaml, but in the values.yaml they are in json format. I'm a bit confused. can you clarify?

and thanks for this, looks cool

Reply:
Hi @cpanato !

We utilize .yaml files for the helm chart configuration, but as YAML allows JSON syntax as well to declare objects, we can within a .yaml file utilize JSON or YAML. In the end, however the object was defined, we will output it with JSON syntax in secret-config.yaml.

```
  # from within templates/secret-config.yaml
  # ...
  # $c is the configJSON object, which first has been merged with the user provided values, and
  # then copied and modified based on chart configuration such as `externalDB.enabled`
  # the "| toJson" part makes the output JSON no matter how it initially was declared
  config.json: {{ $c | toJson | b64enc }}
```

Reply:
So, no matter if .Values.configJSON was in YAML or pure JSON formatting, it is converted to JSON in the end.

Reply:
I figure we may want to decide on sticking to JSON or YAML throughout. JSON is a subset of YAML though so it is just a matter of establishing a convention as it will always really be interpreted as YAML (which may be pure JSON if we decide to use that as a convention for all things related to the config.json file that we generate).